tasks/users_vhosts.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Check if user is OK
+  community.docker.docker_container_exec:
+    container: "{{ rabbitmq_container.name }}"
+    command: 'rabbitmqctl authenticate_user "{{ item.user }}" "{{ item.password }}"'
+  ignore_errors: true
+  register: user_ok
+
+- name: Create user if authentication failed
+  community.docker.docker_container_exec:
+    container: "{{ rabbitmq_container.name }}"
+    command: 'rabbitmqctl delete_user "{{ item.user }}"
```

Comment:
not sure the quoting is right here? or at least in GH interface the syntax highlighting colours look off

Reply:
oh yeah I guess that can be removed